### ISSUE TITLE
Blender imports normal maps to PortableCompressedTexture2D creating huge tscn inherited scenes

### ISSUE THREAD

**Issue by elvisish** [godotengine/godot#86095]
### Tested versions

4.1.3

### System information

Windows 10

### Issue description

Creating an inherited scene from a glb with a normal map imported from Blender creates a huge file due to storing the normal map as PortableCompressedTexture2D.

### Steps to reproduce

1. Import GLB with a normal map assigned in Blender.
2. Create an inherited scene from the glb.
3. The tscn will be around 6x the size.

### Minimal reproduction project (MRP)

N/A (I currently don't have an asset to share but I will try and make one and update this).

**Comment by Calinou** [comment]
- Related to https://github.com/godotengine/godot/issues/34641.

Is the normal map embedded in the GLB, or is it a separate image file?

**Comment by elvisish** [comment]
> * Related to [Materials generated from .gltf file are too big in filesize #34641](https://github.com/godotengine/godot/issues/34641).
> 
> Is the normal map embedded in the GLB, or is it a separate image file?

It came from the GLB, so it should have been embedded in the glb but a lot smaller using Basis Universal:
![image](https://github.com/godotengine/godot/assets/16231628/a6adba25-f823-4630-993b-56ef5150c7d7)


> As a workaround, save the inherited scene to a binary `.scn` file, so that its binary data isn't serialized as text (which is larger and slower).

Oh I didn't know scns were different, I'll keep that in mind.

**Comment by Calinou** [comment]
> Should I save the textures externally now if I want to keep them as TSCNs?

Yes, TSCN/TRES really aren't designed to embed large amounts of binary data.

**Comment by elvisish** [comment]
> > Should I save the textures externally now if I want to keep them as TSCNs?
> 
> Yes, TSCN/TRES really aren't designed to embed large amounts of binary data.

Actually, this is how the non-normal map textures are stored with embedded setting:
![image](https://github.com/godotengine/godot/assets/16231628/58d6592d-5d09-40bc-8589-36eb1c7933cd)

And this is the normal map:
![image](https://github.com/godotengine/godot/assets/16231628/e48c9a84-8294-4671-989d-aef7b8f32632)

Is this a bug?

**Comment by elvisish** [comment]
I just noticed an interesting debugger message:
![image](https://github.com/godotengine/godot/assets/16231628/6db3e95c-4c6f-40c2-8823-34109c1d5009)
